src/engine/ruleset/decoders/fortinet/fortimanager.yml

```diff
@@ -0,0 +1,201 @@
+---
+name: decoder/fortinet-fortimanager/0
+
+metadata:
+  module: fortinet
+  title: Decoder for Fortinet FortiManager Endpoint Security
+  description: Decoder for Fortinet FortiManager Endpoint Security
+  version: [6.0.x, 6.2.x]
+  author:
+    name: Wazuh, Inc.
+    date: 2023/01/20
+  references:
+    - https://docs.fortinet.com/document/fortimanager/7.2.1/log-message-reference/683345/introduction
+
+sources:
+  - decoder/queue-localfile/0
```

Comment:
Can't Fortinet logs come through Syslog?

Reply:
in this case syslog does not apply, we verify it.